src/pages/Home.jsx

```diff
@@ -158,11 +158,11 @@ const Intro = (props) => {
     ref.current.map(clearTimeout);
     ref.current = [];
     set([]);
-    ref.current.push(setTimeout(() => set(['This is ', 'Software', 'Development', 'Club', 'IIT Delhi']), 3000));
-    ref.current.push(setTimeout(() => set(['This is ', 'Development', 'Club', 'IIT Delhi']), 4000));
-    ref.current.push(setTimeout(() => set(['This is ', 'Club', 'IIT Delhi']), 5000));
-    ref.current.push(setTimeout(() => set(['This is ', 'DevClub', 'IIT Delhi']), 8000));
-    ref.current.push(setTimeout(() => set(['This is ', 'DevClub']), 11000));
+    ref.current.push(setTimeout(() => set(['This is ', 'Software', 'Development', 'Club', 'IIT Delhi']), 1000));
```

Comment:
The timings that were hardcoded earlier ensured one by one deletion for 'Software', 'Development' and 'Club'.
I think the previous timings looked better?

Reply:
I think it took a long time for the animation to start, so I reduced the timing of each animation a little. Should I undo this?

Reply:
> The timings that were hardcoded earlier ensured one by one deletion for 'Software', 'Development' and 'Club'.
> I think the previous timings looked better?

It is still one by one deletion, I just removed the latency.